openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -183,6 +184,9 @@ class CourseSectionSequence(CourseContentVisibilityMixin, TimeStampedModel):
     section = models.ForeignKey(CourseSection, on_delete=models.CASCADE)
     sequence = models.ForeignKey(LearningSequence, on_delete=models.CASCADE)
 
+    # Whether or not to show the discussion UI for this particular sequence
+    is_discussable = models.BooleanField(null=False, default=False)
```

Comment:
Sequences don't have is_discussable, Units do.

Reply:
Removed in next series of commits